llvm/lib/Target/AMDGPU/Utils/AMDGPUBaseInfo.cpp

```diff
@@ -34,7 +34,7 @@
 
 static llvm::cl::opt<unsigned> DefaultAMDHSACodeObjectVersion(
     "amdhsa-code-object-version", llvm::cl::Hidden,
-    llvm::cl::init(llvm::AMDGPU::AMDHSA_COV5),
+    llvm::cl::init(llvm::AMDGPU::AMDHSA_COV6),
```

Comment:
Missing test update for this

Reply:
Not really. Nothing changes… I tested it locally

Reply:
The .amdhsa_code_object_version printed for a module with no flag should change. I thought we had a test for the default version already?

Reply:
Actually this is broken, this is the cl::opt which was supposed to be deleted. This isn't actually changing the default, the value comes from the module flag

Reply:
It does things. If we don't set the module flag, it will use this value. For module flag, we don't emit it by default. We only emit it when it is explicitly set by users.

Reply:
> The .amdhsa_code_object_version printed for a module with no flag should change. I thought we had a test for the default version already?

I added a test.